src/server/components/teams.ts

```diff
@@ -32,11 +32,30 @@ export async function getTeams() {
   }
 
   teams = await TeamCollection.find().lean().exec();
+  //sort null programs last
+  sortNullProgramsForTeamsLast(teams);
```

Comment:
api will now sort nulls last (default behavior is to sort nulls first).  This way, Sage will always be last as Sage is by design the only entry with a null program.